### PR TITLE
Registration backend

### PR DESCRIPTION
**Pull request by Laxmankohar**
image

### CODE REVIEW

File: server/controller/newStudentController.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+const newstudentModel = require("../models/newStudent");
+
+const newStudentController = {
+   register:async(req, res)=> {
```
Comment (Gaurav-Verma07) [inline server/controller/newStudentController.js:4]: use controller as functions not objects.

File: server/controller/studentController.js [hunk 2]
```diff
@@ -38,6 +38,7 @@ const Login = async function (req, res) {
     }
 }
 
+
```
Comment (Gaurav-Verma07) [inline server/controller/studentController.js]: remove this extra space

**Comment by Gaurav-Verma07** [comment]
Update `README.md` and _.all-contributorsrc_ CLI with your info.

File: server/controller/studentController.js (hunk 2 above)
Comment (Laxmankohar) [reply server/controller/studentController.js]: Line no 41 has extra space  so, i only need to remove that extra space

Comment (Gaurav-Verma07) [reply server/controller/studentController.js]: Yep, this change on line 41 is not necessary. 🙂

**Comment by Laxmankohar** [comment]
Should i have to make changes in file then after that commit and push again?

**Comment by Gaurav-Verma07** [comment]
> Should i have to make changes in file then after that commit and push again?

Haanji

**Comment by Gaurav-Verma07** [comment]
Update README.md and .all-contributorsrc CLI with your info.

**Pull request merged**